lineapy/utils/analytics/event_schemas.py

```diff
@@ -36,8 +46,7 @@ class LibImportEvent:
 
 @dataclass
 class ExceptionEvent:
-    # TODO: set error_type to be string enums
-    error_type: str
+    error_type: ErrorType
```

Comment:
I would just double check that the JSON event is properly turned into strings (we should have a test harness but a quick manual test is fine for now).